dist/index.html

```diff
@@ -0,0 +1,339 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Catalog</title>
+    <link rel="stylesheet" href="/main.186977be.css">
+  <script src="/main.186977be.js"></script></head>
+
+  <body>
+
+    <header class="header">
+      <a href="home">
+        <img class="img" src="/logo.6023b87e.png" alt="Moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__link" data-qa="hover" href="#laptopsandcomputers">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+
+      <div class="card" data-qa="card">
```

Comment:
```suggestion
    <main>
      <div class="container">

        <div class="card" data-qa="card">
```
for fix test wrap container in main tag

Reply:
Thank You!
It`s so funny ))

I should be more careful ))